go/jsonclient/client.go

```diff
@@ -137,6 +137,17 @@ func (c *JSONClient) PostAndParse(ctx context.Context, path string, req, rsp int
 	return httpRsp, nil
 }
 
+var maxBackoffInterval = 128 * time.Second
+
+func setBackoff(interval time.Duration) (time.Duration, time.Duration) {
```

Comment:
(nit) This function doesn't set anything, so `set...` isn't a great name.  Maybe `calculateBackoff` ?